modules/bluefish/consumer/bluefish_consumer.cpp

```diff
@@ -397,6 +401,10 @@ struct bluefish_consumer : boost::noncopyable
 			if (BLUE_FAIL(blue_->get_card_property32(INVALID_VIDEO_MODE_FLAG, invalidVideoModeFlag)))
 				CASPAR_THROW_EXCEPTION(caspar_exception() << msg_info(" Failed to get invalid video mode flag"));
 
+			// The bluefish HW keyer is going to pre-multiply the RGB with the A. 
+			// This setting results in the correct image coming through when using the 1080_test.tga image.
```

Comment:
It should really be `VIDEO_ONBOARD_KEYER_SET_STATUS_DATA_IS_PREMULTIPLIED` for CasparCG. Notice `IS` instead of `IS_NOT`.

>Hardware Keyer:
>Now using the correct Macro to setup the hw Keyer,
>Be aware the terminology can be confusing as the the macro refers to the source data, not the process to be applied.

The source data **is** premultiplied.

>This has been tested with the casper 1080_test.tga file and produces the correct output.
>Ie. we see the following text in white.
>"Alpha / Key correct: Premultipled
> Linear / Additive"

Unfortunately that file seems to be non useful for this test.